gensim/models/phrases.py

```diff
@@ -715,7 +715,7 @@ def export_phrases(self):
 
         """
         result, source_vocab = {}, self.vocab
-        for token in source_vocab:
+        for token in source_vocab.copy():
```

Comment:
That doesn't sound right. This should be a read-only function ("const"), why are some internal attributes changing?

Reply:
@piskvorky Wow you are right.  Let me investigate further, meanwhile if you have any thoughts please share. It seems like a simple function and it is strange that the dict is changing during runtime.